arangod/Transaction/Methods.cpp

```diff
@@ -3976,8 +3972,7 @@ Future<OperationResult> Methods::replaceInternal(
     f = modifyCoordinator(collectionName, newValue, options,
                           TRI_VOC_DOCUMENT_OPERATION_REPLACE, api);
   } else {
-    OperationOptions optionsCopy = options;
-    f = modifyLocal(collectionName, newValue, optionsCopy,
+    f = modifyLocal(collectionName, newValue, options,
```

Comment:
nit: is it possible to std::move the `options` here into `modifyLocal(...)`? That would safe a copy of the options object, which contains strings with heap-allocations attached to them.